### DISCUSSION TITLE
Custom Nova Action Tag field values not accessible in handle method

### DISCUSSION THREAD

**Discussion by JBiezais** [laravel/nova-issues#5628] [Q&A] [answered]
- Laravel Version: 10.4.1
- Nova Version: 4.25.1
- PHP Version: 8.2.7

### Description:

Creating custom nova action with Tag field, after submitting the form, Tag fields values are not accessible in handle() $field attributes, however, it creates Closure in $field->callbacks, but through this Closure can not gain access to fields values.

**Comment by crynobone** [comment, marked as answer]
`Tag` is strictly used with `BelongsToMany` relationship and isn't supported in Action.
